simple-semantic-search/README.md

```diff
@@ -10,34 +10,47 @@
 # Vespa sample applications - Simple semantic search
 
 A minimal semantic search application: 
-- Query and document text is converted to embeddings by the application. 
-- Search by embedding and/or text match.
+- Query and document text is converted to embeddings by the application using Vespa's [embedder functionality](https://docs.vespa.ai/en/embedding.html#huggingface-embedder).
+- Search by embedding or text match and use [reciprocal rank fusion](https://docs.vespa.ai/en/phased-ranking.html#cross-hit-normalization-including-reciprocal-rank-fusion) to fuse 
+different rankings.
 
 <p data-test="run-macro init-deploy simple-semantic-search">
-Requires at least Vespa 8.54.61.
+This sample app requires at least Vespa 8.54.61.
```

Comment:
This sample app no longer works with 8.54.61: 
`Invalid application package:
Error loading default.default:
Invalid XML according to XML schema, error in services.xml:
attribute "type" not allowed here; expected attribute "bundle" or "class" [7:57], input:
4:
   <container id="default" version="1.0">
5:
6:
       <!-- See https://docs.vespa.ai/en/embedding.html#huggingface-embedder -->
7:
       <component id="e5" type="hugging-face-embedder">`

but again it works with 8.260.15 which is what I happened to have downloaded.

Reply:
I'll remove this line - we usually add it when we have newer features